bcipy/simulator/helpers/types.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+from typing import Optional, List
+
+
+@dataclass
+class InquiryResult:
+    target: Optional[str]
+    time_spent: int  # TODO what does time_spent mean?
+    stimuli: List
+    evidence_likelihoods: List
```

Comment:
Not all evidence produces likelihoods. Language model and button press are probabilities.

Reply:
ok maybe as we incorporate other models, I'll turn this into separate data structure